ui/apps/ui/src/app/components/recommendations/recommendations.component.ts

```diff
@@ -33,11 +33,15 @@ export class RecommendationsComponent implements OnInit {
     this._customRoute.collection$
       .pipe(
         untilDestroyed(this),
-        switchMap((panelId) =>
-          this._recommendationsService
+        switchMap((panelId) => {
+          // TODO: block request if interoperability guidelines
```

Comment:
Personally, I would not use "TODO" here. 
Maybe something like: "IG recommendations are not supported" or no comment would be better?

Reply:
done, 
I thought so